libafl/src/mutators/token_mutations.rs

```diff
@@ -452,6 +452,8 @@ where
 
         let cmps_len = {
             let meta = state.metadata_map().get::<CmpValuesMetadata>();
+            // uncomment to dump cmplog-extracted values
+            // println!("meta: {:x?}", meta);
```

Comment:
This could be a rust feature flag in theory (or a `log::trace()`)

But like this it's fine as well

Reply:
switched to log::trace, as well as log::debug in other place